eventdata/parameter_sources/randomevent.py

```diff
@@ -284,14 +316,23 @@ def generate_event(self):
         # set host name
         event["hostname"] = "web-{}-{}.elastic.co".format(event["geoip_continent_code"], random.randrange(1, 3))
 
-        # determine the raw event size (as if this were contained in nginx log file. We do not bother to
-        # reformat the timestamp as this is not worth the overhead.
-        if self.record_raw_event_size:
+        if self.record_raw_event_size or self.daily_logging_volume:
+            # determine the raw event size (as if this were contained in nginx log file. We do not bother to
+            # reformat the timestamp as this is not worth the overhead.
```

Comment:
not: missing closing parens

Reply:
Thanks. Addressed in https://github.com/elastic/rally-eventdata-track/pull/38/commits/93e06cf2785995dcd25967b261de54c2e8f18a12.